stemcell_builder/stages/rsyslog_config/assets/rsyslog_upstart.conf

```diff
@@ -11,12 +11,6 @@ stop on runlevel [06]
 expect fork
 respawn
 
-pre-start script
```

Comment:
This declaration is removed as `/var/log` never appears on the Google stemcell. There was discussion that `bosh-agent` may be expected to mount this location to ephemeral disk, which does not exist on GCE.